### PR TITLE
Preprocessing scripts

### PR DESCRIPTION
**Pull request by nyxgear**
To applying the preprocessing to **all** the files required, in the `datasets/preprocessed` folder run:

```bash
python3 ./perform_preprocessing.py
```

Further info at [datasets/preprocessed/README.md](https://github.com/polimi-mt-acg/DEBS-2013-soccer-monitoring/blob/58b1d5022ad67114f407a00a20134edd7514a03a/datasets/preprocessed/README.md)

### CODE REVIEW

File: datasets/preprocessed/perform_preprocessing.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+"""
+Apply the preprocessing transformations to all files required.
+"""
+
+import preprocess_metadata
+import preprocess_full_game
```
Comment (leonardoarcari) [inline datasets/preprocessed/perform_preprocessing.py]: How about going for a block:
```py
if __name__ == "__main__":
    process_metadata(source_filepath='./../original/metadata.txt')
    process_full_game(source_filename='./../original/full-game')
```
instead? Maybe we will never need this, but not having executable code upon module import might be desirable.

File: datasets/preprocessed/preprocess_metadata.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                    id3 = reg_match.goalkeeper.group(4).strip()
+                    id4 = reg_match.goalkeeper.group(5).strip()
+                    c = 'PLAYER,{},{},{},{},{},{}'.format(team, name, id1, id2, id3, id4)
+                    print('PARSED', c)
+                    contents.append(c)
+
+    return contents
+
+
+def process_full_game(source_filepath='', target_filepath='metadata'):
```
Comment (leonardoarcari) [inline datasets/preprocessed/preprocess_metadata.py]: I think this should be `process_metadata()`

File: datasets/preprocessed/perform_preprocessing.py (hunk 1 above)
Comment (nyxgear) [reply datasets/preprocessed/perform_preprocessing.py]: Yes! You're totally right, I could have structured this better... Thanks a lot for the suggestion! I'm going to add this :)

File: datasets/preprocessed/preprocess_metadata.py (hunk 2 above)
Comment (nyxgear) [reply datasets/preprocessed/preprocess_metadata.py]: Yes exactly. Thanks again for the given glance!

Done here: [d40034dR137](https://github.com/polimi-mt-acg/DEBS-2013-soccer-monitoring/commit/d40034def62d439d03c014e5e76a15802489451b#diff-9d6b7f0d04b9e005cf5c3655fc3617afR137)

**Pull request merged**